src/page-modules/assistant/client/journey-planner/index.ts

```diff
@@ -7,7 +7,14 @@ export async function nextTripPatterns(
   query: TripQuery,
   cursor: string,
 ): Promise<TripApiReturnType> {
-  const queryString = new URLSearchParams(query).toString();
+  const queryString = Object.keys(query)
+    .map(
+      (key) =>
+        encodeURIComponent(key) +
+        '=' +
+        encodeURIComponent(String(query[key as keyof TripQuery])),
+    )
+    .join('&');
```

Comment:
If this is necessary due to types I would still create is as a named helper function in the bottom of the file:

```ts
function objectToQuery(input: object): string {
  // ... implementation
}
```

Reply:
As far as I can see this is necessary. Makes sense, I'll do that 👍